exe_5_3.py

```diff
@@ -0,0 +1,35 @@
+
+
+def possible_divisions(my_number):
+    """
+    :param my_number: a number for checking if it able to be perfect dish

```

Comment:
There are some grammer mistakes - try to rephrase the documentation.
An easy way would be to define a perfect number at first.